light-prover/prover/circuit_utils.go

```diff
@@ -253,52 +263,99 @@ func LoadVerifyingKey(filepath string) (verifyingKey groth16.VerifyingKey, err e
 
 	return verifyingKey, nil
 }
-func GetKeys(keysDir string, circuitTypes []CircuitType, isTestMode bool) []string {
+func GetKeys(keysDir string, runMode RunMode, circuits []string) []string {
```

Comment:
since this is not code used in circuits, can we move this to `proving_keys_utils.go`?